lms/djangoapps/courseware/views/views.py

```diff
@@ -393,9 +396,13 @@ def course_info(request, course_id):
 
         response = render_to_response('courseware/info.html', context)
         if store_upgrade_cookie:
+            cookie_value = course_id
+            if upgrade_cookie_name in request.COOKIES and course_id not in request.COOKIES[upgrade_cookie_name]:
```

Comment:
So looking at this, it looks like continuously concatenating and setting the expiry on this cookie to 10 days from now, visiting any page with ?upgrade=true will set the expiry for all of the visited pages to 10 days. 

ex. Today I visit course 1 and get a cookie of show_upgrade_notification:Course_1 and an expiry of Feb 26. If I wait 5 days and visit course 2 with ?upgrade=true, I'll now have a show_upgrade_notification:Course_2\Course_1 with an expiry of Mar 3, leaving the Course_1 upgrade banner up for 15 days.

Is that something we're okay with for this test? Would it be preferable to have the course_id as part of the cookie name itself so they all have an individual expiry?